aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -398,7 +409,13 @@ void bgemm<at::BFloat16>(CUDABLAS_BGEMM_ARGTYPES(at::BFloat16)) {
                                   (void*)&falpha, a, CUDA_R_16BF, (int)lda, stridea,
                                   b, CUDA_R_16BF, (int)ldb, strideb,
                                   (void*)&fbeta, c, CUDA_R_16BF, (int)ldc, stridec,
-                                  (int)num_batches, CUDA_R_32F, CUBLAS_GEMM_DEFAULT_TENSOR_OP));
+                                  (int)num_batches,
+#if defined(USE_ROCM) && ROCM_VERSION >= 60000
+                                  CUBLAS_COMPUTE_32F,
```

Comment:
Actually I get an error here: 
no known conversion from 'hipblasLtComputeType_t' to 'hipblasDatatype_t' for 22nd argument

Even if we use the v2 api of hipblasGemmStridedBatchedEx_v2, it takes hipblasComputeType_t on my 6.0.0 header file.

Reply:
There is a hipblaslt API mismatch happening.  This PR was verified against stock ROCm 5.7, as well as a build of ROCm 6.0 that is between RC2 and RC4 -- in anticipation of what the final ROCm 6.0 release should look like.  The tip of tree develop branch of hipblaslt is not yet aligned to our intended API that will be part of 6.0.  If you were to build this hipblaslt PR, it represents what our 6.0 release should look like (and will work with this PR).

https://github.com/ROCmSoftwarePlatform/hipBLASLt/pull/430

Reply:
Just conforming, this PR relies on the latest commit of hipBLASLt (e.g., https://github.com/ROCmSoftwarePlatform/hipBLASLt/pull/430 is committed 5 days ago)? The current public release version of hipBLASLt doesn't work.

Reply:
This PR should not rely on hipblaslt PR 430.  It should work with hipblaslt from its 5.7 release branch, 6.0 release branch, and develop branch.  Unfortunately, all 3 hipblaslt branches have API differences so this PR works around this.